cmd/axelard/cmd/vald/tss/tss.go

```diff
@@ -155,6 +156,7 @@ type Mgr struct {
 	timeoutQueue   *TimeoutQueue
 	Timeout        time.Duration
 	principalAddr  string
+	Keys           *types.ValidatorKeyResponse
```

Comment:
I don't like coupling this to a specific query, can't this just be map[string]Key?